### ISSUE TITLE
Loss and Validation loss is nan

### ISSUE THREAD

**Issue by Mahmood-Hussain** [taipingeric/yolo-v4-tf.keras#13]
nms iou: 0.413 score: 0.3
Epoch 1/50
1125/1125 [==============================] - 408s 349ms/step - loss: 1367.1504 - val_loss: nan

What can be the possible reason for val_loss becoming nan and after first iteration loss itself also becomes nan

**Comment by Mahmood-Hussain** [comment]
> nms iou: 0.413 score: 0.3
> Epoch 1/50
> 1125/1125 [==============================] - 408s 349ms/step - loss: 1367.1504 - val_loss: nan
> 
> What can be the possible reason for val_loss becoming nan and after first iteration loss itself also becomes nan

![Screenshot from 2021-06-28 20-02-16](https://user-images.githubusercontent.com/31275645/123633512-e3bea100-d84b-11eb-800c-38a99f45f905.png)

**Comment by Mahmood-Hussain** [comment]
> > nms iou: 0.413 score: 0.3
> > Epoch 1/50
> > 1125/1125 [==============================] - 408s 349ms/step - loss: 1367.1504 - val_loss: nan
> > What can be the possible reason for val_loss becoming nan and after first iteration loss itself also becomes nan

One more issue is that IT ALWAYS RETURNS SCORE OF 0.25 FOR ALL BOUNDING BOXES

![Screenshot from 2021-06-28 20-30-42](https://user-images.githubusercontent.com/31275645/123636719-c68bd180-d84f-11eb-8104-9e5a35d8aa37.png)

**Comment by maryamxasghari** [comment]
@Mahmood-Hussain did you fix these issues?

**Comment by Mahmood-Hussain** [comment]
@maryamxasghari no I used different source code

**Comment by maryamxasghari** [comment]
@Mahmood-Hussain yolov4-tf? Can you please share the link please? I can not find a good one without issue

**Comment by Mahmood-Hussain** [comment]
@maryamxasghari  This one worked for me without or very few issues. I ran it 2 months ago so forgot whether it had any issue. But this worked for me I also tried many like you: https://github.com/HirataYurina/yoloV4-keras-techi

**Comment by maryamxasghari** [comment]
@Mahmood-Hussain  Thank you
